composer.json

```diff
@@ -4,7 +4,7 @@
     "type": "cakephp-plugin",
     "require": {
         "php": ">=5.4.16",
-        "cakephp/cakephp": "3.5.*"
+        "cakephp/cakephp": "3.6.*"
```

Comment:
This should always be `^3.5` etc, as per semver plugins are compatible to all future Cake minors by default and design.